### PR TITLE
Columbarium redux

### PR DESCRIPTION
**Pull request by davidcarroll**

### CODE REVIEW

File: CmsData/CmsData.csproj [hunk 1]
```diff
@@ -991,7 +992,8 @@
     <Content Include="Migrations\20190520_add_acceptiva_to_gateway_tables.sql" />
     <Content Include="Migrations\20190527_add_preconfigured_acceptiva_details.sql" />
     <Content Include="Migrations\20190620_add_passport_reg_setting.sql" />
-    <Content Include="Migrations\20190827_Update_proc_LedgerIncomeExport.sql" />
+    <Content Include="Migrations\20190827_Update_proc_LedgerIncomeExport - Copy.sql" />
```
Comment (brondavies) [inline CmsData/CmsData.csproj]: Accidental rename/copy?

**Comment by davidcarroll** [comment]
Yes, I just repaired that and pushed. Thanks

**Comment by davidcarroll** [comment]
@brondavies I have no idea why the build is failing. There are no logs or error messages I can see. I don't have a login for teamcity to get any details. It is building fine for me. If you could help me figure that out, I would appreciate it.

**Comment by davidcarroll** [comment]
CodeFactor says to check notice on misc.cs:L0 and I don't see any notice in that file. So I don't know what "complex" code it is complaining about.

**Pull request merged**